pedpy/plotting/plotting.py

```diff
@@ -804,10 +804,10 @@ def _scatter_min_data(
 
         Args:
         axes: The matplotlib axes to plot on.
-        ped_data: DataFrame containing a single pedestrian's data.
+        ped_data: DataFrame containing pedestrian data.
         color: Color of the scatter plot marker.
         """
-        min_data = ped_data[ped_data.frame == ped_data.frame.min()]
+        min_data = ped_data.groupby(ID_COL).apply(lambda x: x[x.frame == x.frame.min()])
```

Comment:
Using `apply` might be slow, as it processes separately. Another approach might be:
```python
        min_data = ped_data.loc[ped_data.groupby(ID_COL)[FRAME_COL].idxmin()]
```

Reply:
I found `idxmin`, but I forgot about `loc` and did not know what to do with it 😅 
Agree that this is better. And also, less formatting, because it is shorter/below the line character limit.